marklogic-data-hub/src/test/java/com/marklogic/hub/flow/impl/StemmingTest.java

```diff
@@ -0,0 +1,102 @@
+package com.marklogic.hub.flow.impl;
+
+import com.marklogic.client.io.StringHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.DatabaseKind;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.flow.RunFlowResponse;
+import com.marklogic.hub.impl.Versions;
+import com.marklogic.hub.job.JobStatus;
+import com.marklogic.mgmt.api.API;
+import com.marklogic.mgmt.api.database.Database;
+import com.marklogic.mgmt.resource.databases.DatabaseManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Assumptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class StemmingTest extends AbstractHubCoreTest {
+
+    Database db;
+    @Autowired
+    FlowRunnerImpl flowRunner;
```

Comment:
Should this go to develop or 5.4-develop - is there any benefit having it on develop? 

If you go to 5.4-develop, you can use the runFlow method that handles everything for you. Actually, that's on develop too. 

In any case, don't use Autowired here - that can causes problems with parallel tests since FlowRunnerImpl isn't a thread-safe class.